README.md

```diff
@@ -347,13 +347,18 @@ This repository also provides one such platforms where contributers come over an
 | [Intellect Quest](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Intellect_Quest) |
 | [Taash_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Taash_Game) |
 | [Number_Guessing_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Number_Guessing_Game) |
+
+| [Shoot_Duck_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Shoot_Duck_Game) |
+| [Modulo_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Modulo_Game) |
```

Comment:
remove the duplicate entry

Reply:
i made changes @ishita-43  mam ..please check it once